.python-version

```diff
@@ -0,0 +1 @@
+2.7.16
```

Comment:
Why are we not using the `python` field under the `[project]` section in the `pyproject.toml` file instead?

Reply:
It's used by pyenv to automatically install and switch to. We could do that also, but that's not in the scope of this pr.

Reply:
Ugh, didn't know about that. Then sure, let's keep it.